requirements/constraints.txt

```diff
@@ -11,9 +11,6 @@
 # Common constraints for edx repos
 -c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt
  
-# Imposed by pytest-cookies 0.5.1 to continue supporting Python 3.4
-arrow<0.14.0
-
 # doc8==1.0.0 requires docutils>=0.19 but sphinx_rtd_theme latest version requires <0.18
 doc8<1.0.0
```

Comment:
Unrelated to this PR, but is this a real issue, and if so, should it move to common constraints?

Reply:
It's a good question.  My experience on side projects is that doc8, docutils, sphinx, and sphinx-rtd-theme have not yet found peace and balance amongst themselves.  Searching the edx and openedx GitHub orgs doesn't find much mention of doc8 in constraints files.  Given that, and the entanglement of packages, I'd leave it alone for now.

`% rg -u -g 'con*.txt' -C=6 -N doc8`

### openedx/edx-cookiecutters/requirements/constraints.txt
```
# Common constraints for edx repos
-c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt

# Imposed by pytest-cookies 0.5.1 to continue supporting Python 3.4
arrow<0.14.0

# doc8==1.0.0 requires docutils>=0.19 but sphinx_rtd_theme latest version requires <0.18
doc8<1.0.0

# These only make sense on Mac
pyobjc-core; sys_platform == 'darwin'
pyobjc-framework-cocoa; sys_platform == 'darwin'
pyobjc-framework-fsevents; sys_platform == 'darwin'
```

### openedx/openedx-events/requirements/constraints.txt
```
# When pinning something here, please provide an explanation of why.  Ideally,
# link to other information that will help people in the future to remove the
# pin when possible.  Writing an issue against the offending project and
# linking to it here is good.
-c common_constraints.txt

# Latest previous version has issues with doc8 because of conflicting docutils constraints
sphinx-book-theme==0.4.0rc1
```

### openedx/event-routing-backends/requirements/constraints.txt
```
# but diff-cover latest has a pin (pluggy<1.0.0a1)
# Using the same version of diff-cover which is being used currently in edx-platform to avoid this conflict.
diff-cover==4.0.0

# greater version failing docs build
sphinx==4.2.0
# Sphinx requires docutils<0.18 && doc8<1.0.0
# This pin can be removed once sphinx constraint is removed.
docutils<0.18
doc8<1.0.0
```

### openedx/edx-django-utils/requirements/constraints.txt
```
# greater version failing docs build
sphinx==4.2.0

# version 1.0.0 requires docutils >0.19 but sphinx@4.2.0 needs docutils<0.18
doc8<1.0.0
```

### openedx/openedx-filters/requirements/constraints.txt
```
# pin when possible.  Writing an issue against the offending project and
# linking to it here is good.

# Common constraints for edx repos
-c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt

# Latest previous version has issues with doc8 because of conflicting docutils constraints
sphinx-book-theme==0.4.0rc1
```